.gitignore

```diff
@@ -1,10 +1,7 @@
-bin
-obj
-packages
+.vs/
+/artifacts/
+/packages/
 Resources/*.ico
 *.Generated.cs
-*.orig
 *.user
-*.suo
-*.exe
-.vs
+Bonsai.Setup.Bootstrapper/vc_redist.*.exe
```

Comment:
We talked about this in our meeting, but for posterity:

.exe: I strongly prefer narrowly defined `.gitignore`s since they help you notice when a mistake is made. (Executables shouldn't be in Bonsai's source tree outside of this one case, so if one appears it might indicate something is messed up in the build system or something.)

`.suo`: Yeah, the loose `suo` hasn't been a thing since like Visual Studio 2013 I think?